frontend/app/assets/javascripts/spree/frontend/views/spree/products/price_filters.es6

```diff
@@ -33,11 +33,15 @@ Spree.ready(function() {
 
   // we have 2 elements for filtering prices - desktop and mobile
   const desktopElement = document.getElementById('filterPriceRangeDesktop')
+  if (!desktopElement) return
```

Comment:
I'm not sure if it's a concern at all, but maybe we shouldn't return early, but do this like here: https://github.com/spree/spree/pull/11021/files#diff-e4fd6659c865684dc415bf38f041c2a35adc03770976e8de6c8a7c500958d831R36

Reply:
@aplegatt please wrap this in if statements like in the linked commit